src/_alfasim_sdk/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.7.0"
+__version__ = "0.7.1"
```

Comment:
```suggestion
__version__ = "0.8.0"
```

Being able to receive a context is definitely a new feature. 👍